http/resp/responder.go

```diff
@@ -254,7 +263,7 @@ func (doer *Responder) Json(w http.ResponseWriter, r *http.Request, opts ...Fn)
 	return nil
 }
 
-/* TODO(dlk): keep?
+/*
```

Comment:
Heads up: https://github.com/xy-planning-network/second-child/blob/9b2437dba2d4edbdf94a7bc5cfeca49a2d5d007d/http/api/api.go#L41

Couple places we need to not nest data

Reply:
Awesome, haven't had the use case in `college-try` but use this on small projects (like where I was doing server-sent events). I'll followup with another PR, get some unit tests in.